README.md

```diff
@@ -1,9 +1,58 @@
 # ion-fusion.github.io
-Source of org-level GitHub Pages
 
-## Local Development
+Source for the Fusion website at: <https://ion-fusion.dev/>
 
-On Mac, you'll need to install Ruby via Homebrew:
+The site is currently built with Jekyll, and deployed via GitHub Pages.
+
+## Development
+
+### Docker
+
+If you have a Docker-compatible CLI ([docker], [podman], [nerdctl], ...) capable of running Debian
+Linux containers, you can build and serve the site via the included [Dockerfile](Dockerfile) with
+invocations like:
+
+```shell
+docker build -t fusion-site .
+```
+
+Or, without even acquiring the sources:
+
+```shell
+docker build -t fusion-site https://github.com/ion-fusion/ion-fusion.github.io.git
+```
+
+After building, you can run the jekyll server with invocations like:
+
+```shell
+docker run -p 4000:4000 fusion-site
+```
+
+Or, to leverage Jekyll's auto-generation and live-reloading:
+
+```shell
+docker run -p 4000:4000 -p 35729:35729 -v "$(pwd):/home/build/fusion-site" fusion-site
```

Comment:
Ah, here's the mount.

Note to self: 35729 is the default live-reload port, which we need to expose outside the container.